tests/container_test.go

```diff
@@ -804,3 +804,44 @@ func stackWithBool(t *testing.T, stack *vm.Stack, v bool) {
 	require.True(t, ok)
 	require.Equal(t, v, res)
 }
+
+func TestPutMeta(t *testing.T) {
+	c, cBal, _ := newContainerInvoker(t, false)
+
+	t.Run("meta disabled", func(t *testing.T) {
+		acc := c.NewAccount(t)
+		cnt := dummyContainer(acc)
+		balanceMint(t, cBal, acc, containerFee*1, []byte{})
+
+		metaInfo, err := stackitem.Serialize(stackitem.NewMapWithValue(
+			[]stackitem.MapElement{{Key: stackitem.Make("cid"), Value: stackitem.Make(cnt.id[:])}}))
+		require.NoError(t, err)
+
+		c.Invoke(t, stackitem.Null{}, "put", cnt.value, cnt.sig, cnt.pub, cnt.token, false)
+		c.InvokeFail(t, "container does not support meta-on-chain", "putObject", metaInfo, nil)
```

Comment:
lets cover all cases:
* invalid meta info (not a map, invalid cid/oid)
* invalid signatures

Reply:
> invalid meta info (not a map, invalid cid/oid)

added

> invalid signatures

can we skip it? it literally just calls `VerifyPlacementSignatures` from the same contract and it is already tested in https://github.com/nspcc-dev/neofs-contract/pull/440 and i have nothing to add to it